### ISSUE TITLE
Survey visualisation pages 

### ISSUE THREAD

**Issue by saschalalala** [Pierre-Sassoulas/django-survey#85]
I don't know if it is in the scope of this project, but for a small project, I am working on, I will have to create some frontend visualizations of the survey state. Of course, the first iteration will be quite focussed on my data, but I think, it would be possible to build a configurable and more generalized version afterwards. Is that something you could think of having inside this project?

At the moment I think, I will build a small json API and a VueJS frontend with chartist.js. This is a quite flexible approach regarding chart types etc from my experience, but of course quite Javascript heavy.

**Comment by Pierre-Sassoulas** [comment]
I think this could be integrated if it's not breaking other URLs/functionalities. Being independent of Vue.js is nice, but having new functionalities is nice too. Also, I don't think scaling is a real issue for the typical user of this lib so I don't mind new dependency too much if they bring value.

Did you check `django-rest-framework` for the small JSON API? This is a pretty neat project, it would probably save you hours of work.

**Comment by saschalalala** [comment]
Yes, I already worked with drf a little and I think I would use it for this functionality too.
